api_tests/base/test_views.py

```diff
@@ -36,6 +37,8 @@
     for patt in urlpatterns:
         VIEW_CLASSES.append(patt.callback.cls)
 
+EXCLUDED_VIEWS = [MoveFileMetadataView, CopyFileMetadataView]
```

Comment:
Minor: since this only applies to `TestApiBaseViews`, it might make sense for this to be a class variable instead of a top-level variable.